### PR TITLE
Documenting how to view `Modelfile`s

### PR DESCRIPTION
**Pull request by jamesbraza**
Upstreaming info from https://github.com/jmorganca/ollama/issues/685:

- Documented tags page in https://ollama.ai/library
- Documented `ollama show --modelfile`

### CODE REVIEW

**Comment by jamesbraza** [comment]
Cc @pdevine @technovangelist

**Comment by jerzydziewierz** [comment]
nice.

File: docs/modelfile.md [hunk 1]
```diff
@@ -42,6 +42,37 @@ INSTRUCTION arguments
 
 ## Examples
 
+### `Modelfile`s in [ollama.ai/library][1]
```
Comment (jmorganca) [inline docs/modelfile.md:45]: Let's move this to the bottom of the doc under `### Viewing the Modelfile for a model`

**Comment by jmorganca** [comment]
Hi @jamesbraza thanks for adding this! Looks pretty good so far, let's just move it a bit further down the doc so that it comes after the example Modelfiles

File: docs/modelfile.md (hunk 1 above)
Comment (jamesbraza) [reply docs/modelfile.md:68]: Okay, just did it, thanks!

**Comment by jamesbraza** [comment]
@jmorganca btw this is ready for your review (or anyone who can review)

**Comment by panckreous** [comment]
This was such a pain point for me initially. Adding documentation like this will be great for all. Thanks, @jamesbraza

**Comment by jamesbraza** [comment]
Hey @jmorganca or @mxyng can I bug for a review here?

**Review by mchiang0610 — APPROVED** [review]

**Pull request merged**